doc/source/cluster/kubernetes/k8s-ecosystem/yunikorn.md

```diff
@@ -0,0 +1,158 @@
+(kuberay-yunikorn)=
+# KubeRay integration with Apache YuniKorn
+
+[Apache YuniKorn](https://yunikorn.apache.org/) is a light-weight, universal resource scheduler for container orchestrator systems. It is created to achieve fine-grained resource sharing for various workloads efficiently on a large scale, multi-tenant, and cloud-native environment. YuniKorn brings a unified, cross-platform, scheduling experience for mixed workloads that consist of stateless batch workloads and stateful services.
+
+KubeRay's Apache YuniKorn integration enables more efficient scheduling of Ray pods in multi-tenant Kubernetes environments.
+
+## Setup
+
+### Step 1: Create a Kubernetes cluster with KinD
+Run the following command in a terminal:
+
+```shell
+kind create cluster
+```
+
+### Step 2: Install Apache YuniKorn
+
+You need to successfully install Apache YuniKorn on your Kubernetes cluster before enabling Apache YuniKorn integration with KubeRay.
+See [Get Started](https://yunikorn.apache.org/docs/) for Apache YuniKorn installation instructions.
+
+### Step 3: Install the KubeRay Operator with Apache YuniKorn support
+
+Deploy the KubeRay Operator with the `--enable-batch-scheduler` flag to enable Volcano batch scheduling support.
+
+When installing KubeRay Operator using Helm, you should use one of these two options:
+
+* Set `batchScheduler.name` to `yunikorn` in your
+[`values.yaml`](https://github.com/ray-project/kuberay/blob/df5577fe1b3f537f36dbda3870b4743edd2f11bb/helm-chart/kuberay-operator/values.yaml#L83)
+file:
+```shell
+# values.yaml file
+batchScheduler:
+    name: yunikorn
+```
+
+* Pass the `--set batchScheduler.name=yunikorn` flag when running on the command line:
+```shell
+# Install the Helm chart with --enable-batch-scheduler flag set to true
+helm install kuberay-operator kuberay/kuberay-operator --namespace ray-system --version 1.2.2 --create-namespace --set batchScheduler.name=yunikorn
+```
+
+### Step 4: Use Apache YuniKorn for batch scheduling
+
+## Example
+
+### Gang scheduling
+
+This example walks through how gang scheduling works with Apache YuniKorn and KubeRay.
+
+First, create a queue with a capacity of 4 CPUs and 6Gi of RAM by editing the ConfigMap:
+
+Run `kubectl edit configmap -n yunikorn yunikorn-defaults`
```

Comment:
Is the ConfigMap `yunikorn-defaults` created by YuniKorn? If so, would you mind mentioning it?